### DISCUSSION TITLE
申请退款：用不了

### DISCUSSION THREAD

**Discussion by liquidzz114** [getlantern/lantern-forum-cn#954] [Q&A]
无法设置系统代理，方法都试过了![微信图片_20241011124553](https://github.com/user-attachments/assets/f60fa2eb-933c-4781-9ffb-647f60c8bc5f)
邮箱18003097959@163.com

**Comment by MaxMKMax** [comment]
@liquidzz114 您好. 好的, 我帮您提交问题上去.
